src/typegraphql.module.ts

```diff
@@ -44,6 +45,25 @@ export class TypeGraphQLModule {
     };
   }
 
+  // Support Apollo Federation
+  static forRootFederation(
```

Comment:
I think it should conform to the NestJS API:
https://docs.nestjs.com/graphql/federation

Reply:
sorry, do you mean to change the options object?

We are using an autogenerated prisma api and your's type-graphql to expose some types to graphql api.

We found this solution to be useful to also indicate that it will be part of a federated api. Do you think we should also add options like orphanedTypes to support all GraphQLFederationModule parameters?

thanks

Reply:
I mean they have `GraphQLModule.forRoot` for standard schema and `GraphQLFederationModule.forRoot` for the federation one.

`typegraphql-nest` wraps `GraphQLModule` into `TypeGraphQLModule`, so I think we should expose `TypeGraphQLFederationModule` which uses `GraphQLFederationModule` under the hood.

Reply:
Perfect!

I commit a change and i think now look good